src/control/server/mgmt_system.go

```diff
@@ -1192,6 +1192,9 @@ func (svc *mgmtSvc) SystemSetProp(ctx context.Context, req *mgmtpb.SystemSetProp
 		if err = proto.Unmarshal(dResp.Body, resp); err != nil {
 			return nil, errors.Wrap(err, "unmarshal PoolSetProp response")
 		}
+		if resp.Status != 0 {
+			return nil, errors.Errorf("SystemSetProp: %d\n", resp.Status)
```

Comment:
suggestion - You can translate the status from DAOS into a useful string using the `Status` type defined in src/control/lib/daos. Something like this:

```suggestion
			return nil, errors.Wrapf(daos.Status(resp.Status), "setting system prop on pool %s", pspr.Id)
```